sunbird-platform-core/common-util/src/main/java/org/sunbird/common/responsecode/ResponseCode.java

```diff
@@ -796,6 +796,9 @@
       ResponseMessage.Key.ERROR_USER_MIGRATION_FAILED,
       ResponseMessage.Message.ERROR_USER_MIGRATION_FAILED),
   invalidIdentifier(ResponseMessage.Key.VALID_IDENTIFIER_ABSENSE,ResponseMessage.Message.IDENTIFIER_VALIDATION_FAILED),
+  mergeeIdRequired(ResponseMessage.Key.MERGEE_ID_MISSING, ResponseMessage.Message.MERGEE_ID_MISSING),
```

Comment:
use same termonology either use mergee or fromAccoundId and update accordingly.

Reply:
used fromAccountId and toAccountId